db/migrate/20150723081851_add_date_to_transactions.rb

```diff
@@ -0,0 +1,14 @@
+class AddDateToTransactions < ActiveRecord::Migration
+  def change
+    add_column :transactions, :date, :datetime
+    reversible do |dir|
+      dir.up do
+        Transaction.with_deleted.where(date: nil).update_all('date = created_at')
```

Comment:
@yunixon you just add column in this migration. How it may be filled? Not dependent on development on production. If you already run migration on production, then you should prepare new migration to fill data, or rollback this (and delete column) and run it again.